src/components/02_atoms/Widgets/TextTextarea.css

```diff
@@ -0,0 +1 @@
+.public-DraftEditor-content { min-height: 110px; }
```

Comment:
Can you move this to emotion? I don't think we're using external css files anywhere else (it's supported because it came with CRA)

Reply:
I'm not sure how I can access the nested component in there

Reply:
It looks like this implementation already uses css files (the need for a css loader in webpack), so there's no js->css api 🤔